### PR TITLE
Documentation for TestContainers

### PR DESCRIPTION
**Pull request by DoodleBobBuffPants**

### CODE REVIEW

File: documentation/database/testcontainers.md
Comment (Lyeeedar) [inline documentation/database/testcontainers.md]: afaik we arent shipping this with the cli

File: documentation/database/testcontainers.md
Comment (Lyeeedar) [inline documentation/database/testcontainers.md]: we should probably list all the jars required, so the final ones (postgres, mysql, etc) and all the other dependencies

File: documentation/database/testcontainers.md
Comment (Lyeeedar) [inline documentation/database/testcontainers.md]: Seeing as we dont ship it, we can remove the 'if' part

**Review by Lyeeedar — CHANGES_REQUESTED** [review]

File: documentation/database/testcontainers.md
Comment (DoodleBobBuffPants) [reply documentation/database/testcontainers.md]: It's the TestContainers JAR we don't ship, but this refers to the actual database driver. TestContainers still requires one of those to be present.

Comment (Lyeeedar) [reply documentation/database/testcontainers.md]: That should be more explicit then, maybe with an example (i.e. 'if using the postgres test container $postgresdependency$, you also need to include etc etc')

**Review by Lyeeedar — APPROVED** [review]

**Pull request merged**
